src/odemis/dataio/test/tiff_test.py

```diff
@@ -355,7 +353,6 @@ def testExportNoWL(self):
                     model.MD_BPP: 12,
                     model.MD_BINNING: (1, 1), # px, px
                     model.MD_PIXEL_SIZE: (1e-6, 2e-5), # m/px
-                    model.MD_WL_POLYNOMIAL: [0], # m, m/px: missing polynomial
```

Comment:
It should be `MD_WL_LIST = []`. That's what the hardware does when the wavelength list is unknown.